### PR TITLE
Vmss: fix creation failure when use unmanaged disk

### PR DESCRIPTION
**Pull request by yugangw-msft**
Found this when went over the live test results. The error is `Parameter 'dataDisks' is not allowed.`.  I also fix a test broken by that `Standard_A4` no longer supports Premium sku.

Moving forward let us still check out live test result daily so we can fix it as early as possible,

//cc: @tjprescott, @troydai 

- [x] The PR has modified HISTORY.rst with an appropriate description of the change (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### Command Guidelines

- [x] Each command and parameter has a meaningful description.
- [x] Each new command has a test.

(see [Authoring Command Modules](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules))

### CODE REVIEW

File: src/command_modules/azure-cli-vm/tests/test_vm_commands.py [hunk 1]
```diff
@@ -593,8 +593,8 @@ def body(self):
             self.resource_group, self.vm_name))
         self._check_vm_power_state('PowerState/deallocated')
         self.cmd('vm resize -g {} -n {} --size {}'.format(
-            self.resource_group, self.vm_name, 'Standard_A4'),
-            checks=JMESPathCheck('hardwareProfile.vmSize', 'Standard_A4'))
+            self.resource_group, self.vm_name, ' Standard_DS2_v2'),
+            checks=JMESPathCheck('hardwareProfile.vmSize', 'Standard_DS2_v2'))
```
Comment (troydai) [inline src/command_modules/azure-cli-vm/tests/test_vm_commands.py]: What's the difference between A2 and DS2_v2 here that will impact the tests?

File: src/command_modules/azure-cli-vm/HISTORY.rst [hunk 2]
```diff
@@ -6,6 +6,7 @@ Release History
 ++++++++++++++++++
 * vm/vmss: lower thread number used for 'vm image list --all' to avoid exceeding the OS opened file limits  
 * diagnostics: Fix a typo in default Linux Diagnostic extension config
+* vmss: fix 'vmss create" failure when use unmanaged-disk
```
Comment (tjprescott) [inline src/command_modules/azure-cli-vm/HISTORY.rst]: Rec:

`vmss create`: fix failure when running with `--use-unmanaged-disk`

File: src/command_modules/azure-cli-vm/tests/test_vm_commands.py (hunk 1 above)
Comment (yugangw-msft) [inline src/command_modules/azure-cli-vm/tests/test_vm_commands.py]: A* is HDD based, but DS* requires SSD which needs to be backed up by premium_* storage 
Likely, the service side changed that you can't downgrade the premium based vm to standard

File: src/command_modules/azure-cli-vm/HISTORY.rst (hunk 2 above)
Comment (yugangw-msft) [inline src/command_modules/azure-cli-vm/HISTORY.rst]: okay

**Comment by troydai** [comment]
Looks good. Approved. Btw, please rebase the pull request because the locations of the tests were moved in https://github.com/Azure/azure-cli/pull/3848.

**Pull request merged**